internal/store/replicationcontroller.go

```diff
@@ -152,6 +154,44 @@ var (
 				}
 			}),
 		},
+		{
+			Name: "kube_replicationcontroller_owner",
+			Type: metric.Gauge,
+			Help: "Information about the ReplicationController's owner.",
+			GenerateFunc: wrapReplicationControllerFunc(func(r *v1.ReplicationController) *metric.Family {
+				labelKeys := []string{"owner_kind", "owner_name", "owner_is_controller"}
+				ms := []*metric.Metric{}
+
+				owners := r.GetOwnerReferences()
+				if len(owners) == 0 {
+					ms = append(ms, &metric.Metric{
+						LabelKeys:   labelKeys,
+						LabelValues: []string{"<none>", "<none>", "<none>"},
+						Value:       1,
+					})
+				} else {
+					for _, owner := range owners {
+						if owner.Controller != nil {
+							ms = append(ms, &metric.Metric{
+								LabelKeys:   labelKeys,
+								LabelValues: []string{owner.Kind, owner.Name, strconv.FormatBool(*owner.Controller)},
+								Value:       1,
+							})
+						} else {
```

Comment:
Do you mind adding a test case for when it is nil, thanks!

Reply:
Sure!